### PR TITLE
Update readme with quickstart steps

### PR DESCRIPTION
**Pull request by muralibasani**
This PR updates the below.

- There is a worker properties file which is still in the name of elasticsearch. Renamed to opensearch
- Updated readme file with quickstart steps
- As this connector is a sink connector, updated the heading.

TODO (new pr)
- Need to change import org.opensearch.common.xcontent.XContentType; to XContentTypeFactory.
- this import org.opensearch.index.VersionType fails
- move integration tests to tests folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,20 @@ curl http://localhost:8083/connector-plugins | jq .
...
 [OpenSearch® Sink Connector Configuration Options](docs/opensearch-sink-connector-config-options.rst)
 
+## QuickStart guide
+
+1. Install and Run Kafka: Ensure Kafka is properly installed and running.
+2. Run Kafka Connect: Make sure Kafka Connect is up and running.
+3. Add OpenSearch Sink to Kafka Connect: Follow the 'How to install' instructions to add the OpenSearch sink connector to Kafka Connect. Example worker config is located here https://github.com/Aiven-Open/opensearch-connector-for-apache-kafka/blob/main/config/quickstart-opensearch.properties
+4. Verify plugin installation : Visit http://localhost:8083/connectors to confirm that the OpenSearch sink connector is listed
+5. Check ACLs (If Enabled): If ACLs are enabled on Kafka, ensure there are no authorization exceptions for the topic and group resources.
+6. Install and Run OpenSearch: Ensure OpenSearch is installed and running. If SSL is enabled, configure the credentials in the connector worker properties.
```
Comment (Claudenw) [inline README.md:85]: Shouldn't this come at the beginning?  It seems like you should have a single first step:

1. Install and configure required software:
    1. Install and Run Kafka (link to install notes)
    2. Install and Run Kafka Connect (link to install notes)
    3. Install and Run OpenSearch (link to install notes) Mention SSL issue

**Review by Claudenw — CHANGES_REQUESTED** [review]
I think the structure should be front to back.  I should not have to stop and install new stuff part way through.  All the required components should be listed at the beginning.

File: README.md (hunk 1 above)
Comment (muralibasani) [reply README.md]: @Claudenw good one. Updated.

File: README.md [hunk 2]
```diff
@@ -75,6 +75,20 @@ curl http://localhost:8083/connector-plugins | jq .
 
 [OpenSearch® Sink Connector Configuration Options](docs/opensearch-sink-connector-config-options.rst)
 
+## QuickStart guide
+
+1. Install and Run Kafka: Ensure Kafka is properly installed and running.
+2. Run Kafka Connect: Make sure Kafka Connect is up and running.
+3. Add OpenSearch Sink to Kafka Connect: Follow the 'How to install' instructions to add the OpenSearch sink connector to Kafka Connect. Example worker config is located here https://github.com/Aiven-Open/opensearch-connector-for-apache-kafka/blob/main/config/quickstart-opensearch.properties
+4. Verify plugin installation : Visit http://localhost:8083/connectors to confirm that the OpenSearch sink connector is listed
+5. Check ACLs (If Enabled): If ACLs are enabled on Kafka, ensure there are no authorization exceptions for the topic and group resources.
```
[the remark this answers is not in the document]
Comment (muralibasani) [reply README.md]: Updated with links.

File: README.md [hunk 3]
```diff
@@ -75,6 +82,21 @@ curl http://localhost:8083/connector-plugins | jq .
 
 [OpenSearch® Sink Connector Configuration Options](docs/opensearch-sink-connector-config-options.rst)
 
+## QuickStart guide
+
+1. Install and Run Kafka: Ensure Kafka is properly installed and running.
+2. Run Kafka Connect: Make sure Kafka Connect is up and running.
+3. Add OpenSearch Sink to Kafka Connect: Follow the 'How to install' instructions to add the OpenSearch sink connector to Kafka Connect. Example worker config is located here https://github.com/Aiven-Open/opensearch-connector-for-apache-kafka/blob/main/config/quickstart-opensearch.properties
```
Comment (Claudenw) [inline README.md:89]: I think you can replace 1. and 2. in this list with "Ensure that the required software is installed and running"
You can then also drop 6 and 7 from this list as it is covered in the required software..

**Review by Claudenw — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (muralibasani) [reply README.md]: ah yes. done.

**Review by Claudenw — APPROVED** [review]
LGTM -- thx for the changes.

**Pull request merged**